__tests__/app.test.js

```diff
@@ -211,3 +211,15 @@ describe("POST /api/technicians/:user_id/reviews", () => {
       });
   });
 });
+
+describe.only("GET /api/user/:user_id/reviews", () => {
+  test("status:200, responds with a reviews from a user with the given user_id", () => {
+    return request(app)
+      .get("/api/users/63ce75449ae462be0adad72a/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toMatchObject([]);
```

Comment:
Could we change this for a test with a user that actually has reviews? If we still want the empty reviews test we can split this into two tests